opentok/opentok.py

```diff
@@ -1599,6 +1600,164 @@ def set_broadcast_layout(self, broadcast_id, layout_type, stylesheet=None, scree
+        """
+        This method starts a render for the specified session.
+
+        :param String 'session_id': The ID of a session (generated with the same `APIKEY` as specified in the URL) which you wish to start rendering into.
+        :param String 'token': A valid OpenTok token with a Publisher role and (optionally) connection data to be associated with the output stream.
+        :param String 'url': A publically reachable URL controlled by the customer and capable of generating the content to be rendered without user intervention.
+        :param Integer 'maxDuration' Optional: The maximum time allowed for the Render, in seconds. After this time, the Render will be stopped automatically, if it is still running.
+        :param String 'resolution' Optional: Resolution of the display area for the composition.
+        :param String 'statusCallbackUrl' Optional: URL of the customer service where the callbacks will be received. 
+        :param Dictionary 'properties' Optional: Initial configuration of Publisher properties for the composed output stream.
+            String name Optional: The name of the composed output stream which will be published to the session.
+        """
+        payload = {
+            "sessionId": session_id,
+            "token": opentok_token,
+            "url": url,
+            "maxDuration": max_duration,
+            "resolution": resolution,
+            "statusCallbackUrl": status_callback_url,
+            "properties": properties
+        }
+
+        logger.debug(
+            "POST to %r with params %r, headers %r, proxies %r",
+            self.endpoints.get_render_url(),
+            json.dumps(payload),
+            self.get_json_headers(),
+            self.proxies,
+        )
+
+        response = requests.post(
+            self.endpoints.get_render_url(),
+            json=payload,
+            headers=self.get_json_headers(),
+            proxies=self.proxies,
+            timeout=self.timeout,
+        )
+
+        if response and response.status_code == 202:
+            return Render(response.json())
+        elif response.status_code == 400:
+            """
+            The HTTP response has a 400 status code in the following cases:
+            You do not pass in a session ID or you pass in an invalid session ID.
+            You specify an invalid value for input parameters.
+            """
+            raise RequestError(response.json().get("message"))
+        elif response.status_code == 403:
+            raise AuthError("You passed in an invalid OpenTok API key or JWT token.")
+        else:
+            raise RequestError("An unexpected error occurred", response.status_code)
+
+
+    def get_render_status(self, render_id):
+        """
+        This method allows you to see the status of a render, which can be one of the following:
+            ['starting', 'started', 'stopped', 'failed']
+
+        :param String 'render_id': The ID of a specific render. 
+        """
```

Comment:
Why this method? Shouldn't there be a `get_render(self, render_id)` method, which returns the entire Render object (including status and other properties)?

Reply:
You're right, this is actually the behaviour of this method so I have renamed as you suggest.